src/Microsoft.AspNet.Authentication.OpenIdConnect/DefaultIdentityModelLoggingListener.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Authentication.OpenIdConnect
+{
+    /// <summary>
+    /// Default logging event listener for wilson event source. It logs the data to the target where other Asp.Net logs go.
+    /// </summary>
+    public class DefaultIdentityModelLoggingListener : EventListener
```

Comment:
`ILogger` is the logging system to use here, not EventListener. As @Tratcher says, `ILogger` is *not* an ASP.NET component. Rather, it is a general purpose logging abstraction used by all DNX-based apps, and even used outside of DNX scenarios (e.g. EF).

There will separately be an EventListener-based implementation of ILogger. We don't want half the system to use ILogger, and half to use EventListener. That's why we have an abstraction: to allow all components to talk to the same abstraction regardless of where the sink is.